src/main/controllers/CitizenHubController.ts

```diff
@@ -40,6 +41,7 @@ export default class CitizenHubController {
     }
 
     const userCase = req.session.userCase;
+    clearTseFields(userCase);
```

Comment:
Why do you need to clearTse fields here?

Reply:
https://tools.hmcts.net/jira/browse/RET-2618  When user press "cancel" he should be redirected to citizen hub page and all the values will be cleared.

Reply:
Happy to leave it, if someone goes back to the citizen hub this would remove the fields but this isn't specified in the tickets how we should deal with it.